tests/unittest_tests.bzl

```diff
@@ -132,6 +132,34 @@ change_setting_with_failure_test = analysistest.make(
     },
 )
 
+####################################
+####### inspect_actions_test #######
+####################################
+def _inspect_actions_test(ctx):
+    """Test verifying actions registered by a target."""
+    env = analysistest.begin(ctx)
+
+    actions = analysistest.target_actions(env)
+    action_output = actions[0].outputs.to_list()[0]
```

Comment:
Ensure the list of actions only has two entries?  check the second action in some way?

(I can see these tests being used as examples also on how to test, so making them more complete seems like a good thing)

Reply:
There's only one registered action here, but I can certainly assert the size of the actions list.

Reply:
> There's only one registered action here, but I can certainly assert the size of the actions list.

I'm confused, `_inspect_actions_fake_rule`?  It has a `ctx.actions.declare_file()` and `ctx.actions.run_shell()`.  I guess that means one of those doesn't end up in the actions list?

Reply:
I don't think `ctx.actions.declare_file` registers an action....